assets/src/dashboard/app/reducer/test/settings.js

```diff
@@ -17,15 +17,22 @@
 /**
  * Internal dependencies
  */
+import { AD_NETWORK_TYPE } from '../../../constants';
 import { ERRORS } from '../../textContent';
 import settingsReducer, { ACTION_TYPES } from '../settings';
 
 describe('settingsReducer', () => {
   const initialState = {
     activePublisherLogoId: null,
     error: {},
-    googleAnalyticsId: null,
+    googleAnalyticsId: '',
```

Comment:
just pulling in the actual initial state for this test.

Reply:
💭 Thoughts about making this a constant and using it in both places?

Reply:
👍  the initial state was actually already exported from the reducer, just hadn't been updated to import. thanks for calling this out